plugins/hu.elte.txtuml.project/src/hu/elte/txtuml/project/wizards/NewTxtUMLModelCreationPage.java

```diff
@@ -139,4 +144,63 @@ protected IStatus packageChanged() {
 		return status;
 	}
 
+	/**
+	 * Checks a package if it belong to an existing model. Searches for a
+	 * package-info.java compilation unit in the package or one of the ancestor
+	 * packages and checks if it has the {@link Model} annotation.
+	 */
+	private static boolean isModelPackage(IPackageFragmentRoot pfRoot, String packageName) throws JavaModelException {
```

Comment:
This branch introduces a new warning compared to master, saying that this function is never used. Looks problematic... Please look into it!